canonical/ubuntu.com
# remove overview and breadcrumbs from bubble navigations

## Done

- Simplified the subnav across the site, so that:
  - there are no more than 6 sub nav items present on any bubble
  - the "Overview" item is not visible
  - there are no sub-subnav/breadcrumb versions of the bubble nav, such as this (https://ubuntu.com/legal/terms):
![image](https://user-images.githubusercontent.com/2376968/185620578-ecf4234e-7d3e-4d6e-a10a-d40052024383.png)


## QA

- [Open this spreadsheet](https://docs.google.com/spreadsheets/d/1YeTZP9sr2WUX2-N7clirZtGuARD0PMirHk0Ty-WUKU0/edit#gid=987317135)
- Visit https://ubuntu-com-11957.demos.haus/
- Work through the list of bubbles, see that any item in the spreadsheet that has a red title is no longer visible on the site in the subnav
- See that the subnav does not look broken

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5237
